utils/supabase-admin.ts

```diff
@@ -285,7 +285,7 @@ const onPaid = async (document_id: string, customer_email: string) => {
     for (const lead in leadData) {
       const { id, email } = leadData[lead];
 
-      const response = await leap.workflowRuns.workflow({
+      leap.workflowRuns.workflow({
```

Comment:
the is the only important part, the rest of the changes is weird formatting (in a good way) that my editor decided to do :)